Holy_locket.WebAPI/Program.cs

```diff
@@ -17,6 +23,9 @@
     .AllowAnyHeader();
 }));
 
+Test test = new Test();
+test.Start();
+
```

Comment:
is it required in the remote repo? if it is testing only local db connection, please remove it and don't push comments to remote